src/client/containers/404Page/404Page.styles.css

```diff
@@ -0,0 +1,11 @@
+.page404-text {
+  text-align: center;
+  font-weight: 50px;
+  padding: 10rem 0;
+  color: #bdbdbd;
+}
+
+.text-404 {
+  font-size: 100px !important;
```

Comment:
Why the use of `!important`? Please remove it.

Reply:
removed